docs/source/frida_bindings.rst

```diff
@@ -67,6 +67,98 @@ Analysis
     :members: QBDI.prototype.getInstAnalysis, QBDI.prototype.getInstMemoryAccess, QBDI.prototype.getBBMemoryAccess
     :member-order: alphabetical
 
+.. js:class:: InstAnalysis
+
+  Object describes the Analysis of the current instruction
+
+  .. js:attribute:: address
+
+      Instruction address (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: affectControlFlow
+
+      True if instruction affects control flow (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: disassembly
+
+      Instruction disassembly (if ANALYSIS_DISASSEMBLY)
+
+  .. js:attribute:: instSize
+
+      Instruction size (in bytes) (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: isBranch
+
+    True if instruction acts like a ‘jump’ (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: isCall
+
+    True if instruction acts like a ‘call’ (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: isCompare
+
+    True if instruction is a comparison (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: isPredicable
+
+    True if instruction contains a predicate (~is conditional) (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: isReturn
+
+    True if instruction acts like a ‘return’ (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: mayLoad
+
+    True if instruction ‘may’ load data from memory (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: mayStore
+
+    True if instruction ‘may’ store data to memory (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: mnemonic
+
+    LLVM mnemonic (if ANALYSIS_INSTRUCTION)
+
+  .. js:attribute:: module
+
+    Instruction module name (if ANALYSIS_SYMBOL and found)
+
+  .. js:attribute:: operands
+
+    Structure containing analysis results of an operand provided by the VM (if ANALYSIS_OPERANDS)
+
+  .. js:attribute:: symbol
+
+    Instruction symbol (if ANALYSIS_SYMBOL and found)
+
+
+.. js:class:: MemoryAccess
+
+  Object describes a memory access.
```

Comment:
Object that describes ?

Reply:
It was originalyly `Object described a memory access.` that I changed to `[this] Object describes a memory access.` ("this" is implied here, but we could also used your formulation.

Reply:
fix